priv/scripts/inject_erlang_cookie.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cat ./rel/config.exs | sed "s/{ERLANG_COOKIE}/`cat erlang.secret.cookie`/g" > ./rel/config.exs
```

Comment:
Doesn't this already happen if you use `${ERLANG_COOKIE}` as the value and `REPLACE_OS_VARS=true` in the environment?

Reply:
Oh man. We had the `REPLACE_OS_VARS=true` BELOW the place it was needed.

I'm removing this script and updating the readme

Reply:
Fixed in fdfeaf7665275b6df7339a3ecc88dee62db767a3